### ISSUE TITLE
nvda does not read the border and shader information in a word document

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#9050]
### Steps to reproduce:
Create a document in word select a portion of text and apply a border and a shading to the selected text.
### Actual behavior:
When you focus on the text that received the border and the shading and if you press nvda + f so that nvda announces the formatting, nvda does not report the information about which border was applied, the style of the border applied, the color of the border applied , on which sides of the text the border is applied.
Nvda also does not speak if there is a text shading, and does not also announce the color and intent of the shading applied.
### Expected behavior:
Nvda could announce all aspects of edge and shadowing reported above.
#### NVDA installed/portable/running from source:
install
#### NVDA version:
alfa 16387
#### Windows version:
10 17763.168
#### Name and version of other software in use when reproducing the issue:
office 2013
#### Other information about your system:
At the moment I do not have office 2016 or 2019 to test
#### Does the issue still occur after restarting your PC?
yes
#### Have you tried any other versions of NVDA?
no

**Comment by Adriani90** [comment]
Closing as duplicate of #2511.
